go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -566,7 +566,14 @@ func (bnc *BaseNetworkController) addLogicalPortToNetwork(pod *kapi.Pod, nadName
 	}
 
 	// CNI depends on the flows from port security, delay setting it until end
-	lsp.PortSecurity = addresses
+	if !bnc.NetInfo.DisablePortSecurity() {
+		lsp.PortSecurity = addresses
+	} else {
+		// if we want to allow traffic from other workloads through, we must add
+		// `unknown` to the list of addresses
+		const allowUnknownAddresses = "unknown"
+		lsp.Addresses[0] = lsp.Addresses[0] + " " + allowUnknownAddresses
```

Comment:
@dceara hoping for your OVN wisdom ...

From my limited understanding, it doesn't make much sense to have IPs & unknown in the address attributes ... 

Is setting the entire `addresses` attribute to `unknown` better than what I'm doing ? Without thinking much through, I'd say performance wise it would be slightly preferable ...

Reply:
@maiqueb LSP.addresses is used by OVN also allow responding to ARP/ND requests locally, on the hypervisor, instead of tunneling to other hypervisors in case the logical switch is distributed.  However, if I understand correctly your case, the logical switch is already only available on a single node so it might be fine to just use "unknown".

Reply:
hm, no the logical switch can be distributed.

I'm implementing this feature for all 3 topologies:
- layer3: logical switch per node, inter-connected by a router
- layer2: cluster-wide logical switch
- localnet: cluster-wide logical switch with a localnet port; OVN bridge mappings configured out-of-band

For the layer2 and localnet topologies the logical switch is cluster-wide. 
IIUC you recommend the current approach, right ? I.e. having in `lsp.addresses` the mac, the known IPs, plus unknown ? 
@dceara 

Thanks for your input.

Reply:
> For the layer2 and localnet topologies the logical switch is cluster-wide.
> IIUC you recommend the current approach, right ? I.e. having in `lsp.addresses` the mac, the known IPs, plus unknown ?
> @dceara

@maiqueb Yes, that's right, it will avoid tunneling ARP packets.  It's better to leave the known IPs in the lsp.addresses field.

Reply:
Perfect, thank you :)